### ISSUE TITLE
Do not open login shell (-s & -ss option)

### ISSUE THREAD

**Issue by greymd** [greymd/tmux-xpanes#93]
Unify all the work logs about following features.

```
  -s: Speedy mode: Run commands promptly. But panes are not interactive.
  -ss: Speedy mode and close windows automatically when process ends.
```

`-s` option  is came from "speedy" or "sh" or something.
I believe this feature solves both #89  #90 .

**Comment by greymd** [comment]
Memo: tmux behavior of `split-window` command.
https://github.com/tmux/tmux/blob/7448b38327f856dae6ce915eae63f7f6c3f036e0/window.c#L983

>		 * If given one argument, assume it should be passed to sh -c;
>		 * with more than one argument, use execvp(). If there is no
>		 * arguments, create a login shell.

https://github.com/tmux/tmux/blob/master/cmd-split-window.c#L122

```c
	shell = options_get_string(s->options, "default-shell");
	if (*shell == '\0' || areshell(shell))
		shell = _PATH_BSHELL;
```

https://github.com/tmux/tmux/blob/master/compat.h#L65

```c
#define	_PATH_BSHELL	"/bin/sh"
```

If `default-shell` is defined, it is used with `execl`.
`-c` is also hardcoded.

https://github.com/tmux/tmux/blob/master/cmd-split-window.c#L122
```
			execl(wp->shell, argv0, "-c", first, (char *)NULL);
```

Otherwise `/bin/sh` is used with `execvp`.

That's why following command does not open login shell.

```
$ tmux split-window "ls; sleep 1"
```

Memo: Document about exec series is here.
https://gist.github.com/fffaraz/8a250f896a2297db06c4

**Comment by greymd** [comment]
If you want "to execute commands as is, promptly, on the existing pane", then `-sex` option is required.
Like this.

```
$ xpanes -sex 'echo '{01..15}
```

**Comment by greymd** [comment]
I am now working on this feature on develop-3.1.0 branch and touch upon the new options.
Finally, I am pretty sure that suppressing closing pane should be default to improve usability.

Let me change the specification of this feature like following.
Abolish `-a` option. set `-ss` option instead.

Before:
```
-s: option:  Speedy mode. Just execute command. Do not make panes interactive.
-a: keep alive option (do not close pane when the command finished when -s option)-s: option:  Speedy mode. Just execute command. D
```

After:
```
  -s: Speedy mode: Run commands promptly. But panes are not interactive.
  -ss: Speedy mode and close windows automatically when process ends.
```

There are three reasons:
1. `xpanes --help` displays all the options alphabetical order.
If `-a` option is going to be placed upper part of the --help/man page,
it is difficult for users to notice the relationship between new option and `-s` option.

2. It is common to use same letters sequentially in Unix commands.
i.e: `ssh -vvv` option.

3. easier to type than `-a`

**Comment by greymd** [comment]
This feature is released at [v3.1.0](https://github.com/greymd/tmux-xpanes/releases/tag/v3.1.0)

**Comment by greymd** [comment]
![movie](https://user-images.githubusercontent.com/3874767/43615818-6489f350-96b1-11e8-8c32-f89f591c89f9.gif)
